cime_config/buildnml

```diff
@@ -127,7 +127,7 @@ def _create_drv_namelists(case, infile, confdir, nmlgen, files):
     elif case.get_value("RUN_TYPE") == "branch":
         config["run_type"] = "branch"
 
-    config['wav_ice_coupling'] = config['COMP_WAV'] == 'ww3dev' and config['COMP_ICE'] == 'cice'
+    config['wav_ice_coupling'] = 'ww3' in config['COMP_WAV'] and config['COMP_ICE'] == 'cice'
```

Comment:
Can we please make this backwards compatible with the previous version by adding 
config['COMP_WAV'] == 'ww3dev' or config['COMP_WAV'] == 'ww3'
That way we can use this in NorESM for now.

Reply:
I did that.  ww3 is a substring of ww3dev and so 'ww3' in config['COMP_WAV'] works for both.

Reply:
Sorry - I did not catch that. Looks good.